components/server/src/api/configuration-service-api.ts

```diff
@@ -58,6 +62,8 @@ export class ConfigurationServiceAPI implements ServiceImpl<typeof Configuration
         private readonly userService: UserService,
         @inject(ContextService)
         private readonly contextService: ContextService,
+        @inject(PrebuildManager)
+        private readonly prebuildManager: PrebuildManager,
```

Comment:
Uh, this works without cyclic dependencies? We got lucky I guess :laughing:

Reply:
Resolved in [`7bd89de` (#20228)](https://github.com/gitpod-io/gitpod/pull/20228/commits/7bd89de711bd3c0950a1f83b3ff7143cf3502d57)